mavros/src/lib/uas_stringify.cpp

```diff
@@ -176,10 +180,10 @@ static inline std::string str_mode_px4(uint32_t custom_mode_int)
 static inline bool is_apm_copter(UAS::MAV_TYPE type)
 {
 	return type == UAS::MAV_TYPE::QUADROTOR ||
-	       type == UAS::MAV_TYPE::HEXAROTOR ||
-	       type == UAS::MAV_TYPE::OCTOROTOR ||
-	       type == UAS::MAV_TYPE::TRICOPTER ||
-	       type == UAS::MAV_TYPE::COAXIAL;
+		   type == UAS::MAV_TYPE::HEXAROTOR ||
```

Comment:
Thia still doesn't seem right...

Reply:
Sorry, Sublime changed some spaces to indents when it shouldn't have.